app/component/MainMenu.js

```diff
@@ -16,6 +16,11 @@ import intializeSearchContext from '../util/DTSearchContextInitializer';
 
 function MainMenu(props, { config, intl }) {
   const [countries, setCountries] = useState(props.countries);
+  const appBarLinkHref =
+    config.appBarLink.alternativeHref &&
```

Comment:
Tämä ehkä menee siistimmin näin:

Href  = config.appBarLink.alternativeHref?.[config.appBarLink.alternativeHref] || config.appBarLink.href;